adm/src/commands/batch.rs

```diff
@@ -64,11 +72,11 @@ fn run_list_command<'a>(args: &ArgMatches<'a>) -> Result<(), CliError> {
         }
     } else if format == "json" || format == "yaml" {
         let structured_output_batches = batches
-            .map(|item: Result<Batch, _>| item.map(|batch| StructuredOutputBatch::from(&batch)))
-            .collect::<Result<Vec<StructuredOutputBatch>, SawtoothClientError>>()
+            .map(|item: Result<Batch, _>| item.map(|batch| DisplayBatchListItem::from(&batch)))
+            .collect::<Result<Vec<DisplayBatchListItem>, SawtoothClientError>>()
```

Comment:
Looks like since the duplicate `SawtoothClientError` import is being removed, this should be changed to `ClientError`